### ISSUE TITLE
Some questions about the paper.

### ISSUE THREAD

**Issue by yangyuya** [xzz777/SCTNet#5]
Hi,

Very nice job! But I have some questions when I'm reading the paper:

1、During training, SegFormer is pretrained or training from scratch? If training from scratch，can SegFormer reach the performance in the paper（About 46.5 for SegFormer-B2 in ADE20k.）?
2、For the results in Table5, does DDRNet-23+SIAM mean that  DDRNet+SIAM+SegFormer-B3? And which brach in DDRNet does the Backbone Feature Alignment Block use, 1/8+1/8 feature or 1/32+1/64 feature？Why is the improvement of SegNext-T slim either？It's also a single-branch model.  Maybe the reason is the small gap between the performance of student and teacher model？

Looking forward to your reply and code releasing！

**Comment by xzz777** [comment]
Thank you for your attention to our work, and apologies for the delayed response.

1) During training, SegFormer is pretrained. If used properly, theoretically, joint training can improve the accuracy performance of both models. However, joint training requires significant GPU memory and long iterations, as discussed in Appendix D Discussions (Knowledge Distillation for Isomeric Models). If you are interested in such setting, you can read the reference paper "A Good Student is Cooperative and Reliable: CNN- Transformer Collaborative Learning for Semantic Segmentation"(ICCV23), which is also a nice job, but unfortunately, the authors have not released the code until now.

2a) Yes, DDRNet-23+SIAM means DDRNet+SIAM+SegFormer-B3. We use 1/32+1/64 features, and the same applies to SeaFormer. For bilateral networks, we use the features of their semantic branches, which aligns with our approach of treating the transformer branch as a train-only semantic branch.

2b) For SegNext, the small gap is a possible reason we haven't explored. For fair ablation, we use the same teacher model.  Similarly, in Table 10 (Appendix), we also control for the accuracy of all teacher models (for those with high accuracy, we use intermediate checkpoints with accuracy similar to SegFormer-B3).

PS: Tomorrow, I will update the weight conversion script for teacher models. If you are interested in the performance of SCTNet and other methods under a larger transformer teacher, such as SegFormer B4 or B5, you can download the specified model from the mmseg model zoo and conduct relevant experiments after weight conversion.
